backend/app2/llm/adapters/base_adapter.py

```
@@ -0,0 +1,140 @@
+from abc import ABC, abstractmethod
+from typing import List, Any, AsyncGenerator, Dict, Optional, Sequence
```

Ruff (0.8.2)

`typing.AsyncGenerator` imported but unused

Remove unused import: `typing.AsyncGenerator`

(F401)